app/controllers/email_controller.rb

```diff
@@ -26,7 +26,7 @@ def unsubscribe
   def perform_unsubscribe
     RateLimiter.new(nil, "unsubscribe_#{request.ip}", 10, 1.minute).performed!
 
-    key = UnsubscribeKey.find_by(key: params[:key])
+    key = UnsubscribeKey.includes(:user).find_by(key: params[:key])
```

Comment:
Am I right in saying this is just a perf optimization, with no behaviour change?

Reply:
Yeah, it technically isn't an improvement because it will do 2 SQL requests no matter which way we do it, but at least, with the "includes" it's explicit and matches the rails way of "side loading"

Reply:
Like the other comment, ideally I think this would be moved into a separate `DEV:` PR. It's unrelated to the RFC8058 implementation, which makes review & future `git blame` very confusing.

Reply:
@davidtaylorhq just split the changes in two commits